README.md

```diff
@@ -1,6 +1,13 @@
 # Shortest
 **[shortest.com](https://shortest.com)**
 
+![GitHub Repo stars](https://img.shields.io/github/stars/anti-work/shortest?style=social)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[![npm version](https://img.shields.io/npm/v/@antiwork/shortest)](https://www.npmjs.com/package/@antiwork/shortest)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[![Downloads](https://img.shields.io/npm/dm/@antiwork/shortest)](https://www.npmjs.com/package/@antiwork/shortest)
```

Comment:
Personally think we should copy Firecrawl's approach, keeping the stars separate and having a CTA to like. I don't think including the stars by itself helps as it's already above. Wdyt?

Reply:
Perhaps we could add the Logo at the top and show the link to the website like how Firecrawl does. Just double checking would you like to add the rest of the stuff like Firecrawl or the Logo and the demo video is enough for now? 

![image](https://github.com/user-attachments/assets/7e98cf6d-8529-4b73-811c-1aab00e326d2)

Reply:
We could add the logo as it's already in the repo. Nextjs does similarly